src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  &:nth-last-child(n) {
+    margin-right: 4px;
+  }
+
+  .stars--4 &:not(:nth-child(n+5)) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
you should reuse all code for stars (for each counts of stars)

Reply:
What do you mean?